network-api/networkapi/wagtailcustomization/static/wagtailadmin/css/custom-fix.css

```diff
@@ -1,5 +1,5 @@
 /*
-  To address bug from Wagtail v3
+  To address bug from Wagtail v3+
```

Comment:
Keeping this portion of custom CSS because this bug is still happening on Wagtail v4.